src/providers/in-app-browser/card.ts

```diff
@@ -207,17 +207,26 @@ export class IABCardProvider {
          * */
 
         case 'pairing':
-          const { secret } = event.data.params;
           // generates pairing token and also fetches user basic info and caches both
           this.bitpayIdProvider.generatePairingToken(
-            secret,
+            event.data.params,
             (user: User) => {
               if (user) {
                 this.getCards();
                 this.user.next(user);
               }
             },
-            err => this.logger.debug(err)
+            () => {
```

Comment:
Sometimes is a good idea to add the `e` variable here and show in console.log, in spite of we show another message to the user

Reply:
added a log.error!